packages/lib-classifier/src/components/Classifier/components/SubjectViewer/helpers/getViewerComponent/getViewerComponent.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import RasterViewer from '../../components/RasterViewer'
+import convertLocationToObject from './convertLocationToObject'
+
+const viewers = {
+  image: {
+    jpeg: RasterViewer,
+    jpg: RasterViewer,
+    png: RasterViewer
```

Comment:
Why are the viewers being mapped to specific subtypes? Won't this cause maintenance difficulties as the number of supported file types grows?

Reply:
The mime type for SVG is `image/svg+xml`, so there will need to be _some_ subtype mapping

Reply:
I'm confused. We don't have subtype mapping in the existing code, but we support SVG images.

Reply:
OHHHH I know why, the entire Classifier is an SVG so that gets plugged straight in (and presumably a conditional to add in an `image` tag for non svg images)

Reply:
We could bite the bullet and use SVG to render all images, regardless. It makes zooming, panning and drawing a lot easier but could be overkill for a project like Galaxy Zoo.

Reply:
I'd be happy with that, it's one less path for the code to go down.